### PR TITLE
TST: Fixtures and test clean up for test_dissolve.py

### PR DESCRIPTION
**Pull request by cr458**
Removed testing class in `test_dissolve.py`, using fixtures instead.

### CODE REVIEW

File: geopandas/tests/test_dissolve.py [hunk 1]
```diff
@@ -8,78 +10,83 @@
...
+    nybb_polydf = read_file(nybb_filename)
+    nybb_polydf = nybb_polydf[['geometry', 'BoroName', 'BoroCode']]
+    nybb_polydf = nybb_polydf.rename(columns={'geometry': 'myshapes'})
+    nybb_polydf = nybb_polydf.set_geometry('myshapes')
+    nybb_polydf['manhattan_bronx'] = 5
+    nybb_polydf.loc[3:4, 'manhattan_bronx'] = 6 
+    return  nybb_polydf
+
+@pytest.fixture
+def test_merged_shapes(nybb_polydf):
```
Comment (jorisvandenbossche) [inline geopandas/tests/test_dissolve.py]: Maybe better don't name the fixtures with `test_` to not confuse with actual test?

File: geopandas/tests/test_dissolve.py [hunk 2]
```diff
@@ -8,78 +10,83 @@
...
+
+    collapsed = [others.geometry.unary_union,
+                 manhattan_bronx.geometry.unary_union]
+    test_merged_shapes = GeoDataFrame(
+        {'myshapes': collapsed}, geometry='myshapes',
+        index=pd.Index([5, 6], name='manhattan_bronx'))
+
+    return test_merged_shapes
+
+@pytest.fixture
```
Comment (jorisvandenbossche) [inline geopandas/tests/test_dissolve.py]: Everywhere in the file: can you have two blank lines between functions ? (PEP8)

File: geopandas/tests/test_dissolve.py (hunk 1 above)
Comment (cr458) [reply geopandas/tests/test_dissolve.py]: Yup! Fixed.

File: geopandas/tests/test_dissolve.py (hunk 2 above)
Comment (cr458) [reply geopandas/tests/test_dissolve.py]: Fixed.

**Comment by cr458** [comment]
Looks like I stupidly branched from my the test_dataset branch rather than the original master branch. Would you like me to rebase from the tip of the current master branch? I think that should get rid of the commits.

**Comment by jorisvandenbossche** [comment]
Rebasing was perfect in this case. 

Thanks a lot!

**Pull request merged**